packages/components/src/atoms/Price/Price.tsx

```diff
@@ -54,7 +54,6 @@ const Price = forwardRef<Omit<HTMLSpanElement, 'children'>, PriceProps>(
     ref
   ) {
     const formattedPrice = formatter(value, variant)
-
```

Comment:
I didn't mean to remove this line 😅 could you re-add it without identation pls?

Reply:
Sure! haha sorry I was just committing the suggestion but I´ll fix this soon :)